.github/workflows/check-advisories.yml

```diff
@@ -52,8 +39,8 @@ jobs:
           echo "code-hash=$code_hash" >> "$GITHUB_OUTPUT"
   check_advisories:
     name: Invoke check-advisories workflow
-    needs: [tools_changed, advisories_changed, code_hash]
-    if: ${{ needs.tools_changed.outputs.should_skip == 'true' && needs.advisories_changed.outputs.should_skip != 'true' }}
```

Comment:
That's work duplication as advisories are checked whenever `hsec-tools` change

Reply:
I removed this commit.